pkg/util/util.go

```diff
@@ -100,3 +102,28 @@ func PatchService(ctx context.Context, client clientset.Interface, cur, mod *v1.
 
 	return nil
 }
+
+func GetAZFromTopology(topologyKey string, requirement *csi.TopologyRequirement) string {
+	var zone string
+	var exists bool
+
+	defer func() { klog.V(4).Infof("detected AZ from the topology: %s", zone) }()
```

Comment:
I think the zone log better to be V(1) so we know what's selected for trouble shooting

Reply:
will do